_sass/_base.scss

```diff
@@ -20,6 +20,12 @@
     }
   }
 
+  .yandex-metrika-counter {
```

Comment:
Why do you add this? What problem solve this styles code?

Reply:
Your webpage is using inline CSS styles! One of the problem from https://seositecheckup.com/seo-audit/betterdocs.herokuapp.com#common-seo-issues